libkbfs/disk_block_cache.go

```diff
@@ -589,10 +588,9 @@ func (cache *DiskBlockCacheLocal) evictUntilBytesAvailable(
 }
 
 // Put implements the DiskBlockCache interface for DiskBlockCacheLocal.
-func (cache *DiskBlockCacheLocal) Put(ctx context.Context, tlfID tlf.ID,
-	blockID kbfsblock.ID, buf []byte,
-	serverHalf kbfscrypto.BlockCryptKeyServerHalf,
-	_ DiskBlockCacheType) (err error) {
```

Comment:
why did this used to take an unused cache type?

Reply:
Because it had to satisfy the `DiskBlockCache` interface before, and now it doesn't have to anymore.